### ISSUE TITLE
Option to hide Alt text by default when viewing media

### ISSUE THREAD

**Issue by DukeOFprunesOriginal** [tuskyapp/Tusky#4669]
### Pitch

Alt text size is very generous. Many videos or animated gifs start playing behind a wall of it. Alt text should optionally be hidden by default for those who do not wish to see it

### Motivation

Everyone. It's fine to keep it on by default dor accessibility but not everyone wants or needs to see it for every image or video they view

**Comment by connyduck** [comment]
I don't like adding more options. Instead we should think about a better layout where one can (for example) easily swipe the text in or out of view.

**Comment by fsmv** [comment]
I think the simplest possible solution to this would be to make it so tapping on the screen dismisses the controls.

I noticed that videos without alt text actually already work this way, it just doesn't work when there is alt text.

In my opinion the best UX would be having a separate button in the video player controls toolbar that opens the alt text display. So it wouldn't display the alt text by default, only if you press the button.

**Comment by fsmv** [comment]
Here's a video with alt text: https://mstdn.social/@yurnidiot/113801266467466095

and a video without alt text: https://infosec.exchange/@tracketpacer/113801081640670982

Seems to me it's a bug that videos without alt text don't allow tap to dismiss @connyduck

**Comment by JDeeth** [comment]
I agree with the initial comment - alt-text should be on by default for accessibility. But for myself I'd like alt-text hidden by default on my device - could this be a situation where providing an option is worthwhile?

It wouldn't matter so much when #4854 is fixed and I can easily dismiss alt-text on videos again. (The reason I logged in and found the repo today is, if I want to show a funny video to my partner without alt-text spoiling the gag, I have to open the post in the mobile browser, which is a bit clunky)
